spec/models/user_spec.rb

```diff
@@ -1,5 +1,44 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  user = User.new(name: 'Kossi', bio: 'Formal Civil Engineer', post_counter: 0, photo: 'https://avatars.githubusercontent.com/u/74506933?v=4')
+
+  before(:each) { user.save }
+
+  it 'validates the presence of the name' do
```

Comment:
- [ ] It would be better if we write validation tests for our models. For example; 
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

Reply:
Fixed